tests/test_apigateway/test_apigateway.py

```diff
@@ -959,6 +959,133 @@ def test_delete_authorizer():
     )
 
 
+@mock_apigateway
+def test_create_request_validator():
+    ID = "id"
+    NAME = "name"
+    VALIDATE_REQUEST_BODY = "validateRequestBody"
+    VALIDATE_REQUEST_PARAMETERS = "validateRequestParameters"
+    PARAM_NAME = "my-validator"
+    client = boto3.client("apigateway", region_name="us-west-2")
+    response = client.create_rest_api(name="my_api", description="this is my api")
+    api_id = response[ID]
+    response = client.create_request_validator(
+        restApiId=api_id,
+        name=PARAM_NAME,
+        validateRequestBody=True,
+        validateRequestParameters=True,
+    )
+
+    response.pop("ResponseMetadata")
+    response.pop("id")
+    response.should.equal(
+        {
+            NAME: PARAM_NAME,
+            VALIDATE_REQUEST_BODY: True,
+            VALIDATE_REQUEST_PARAMETERS: True,
+        }
+    )
+
+
+@mock_apigateway
+def test_get_delete_request_validators():
+    ID = "id"
+    NAME = "name"
+    VALIDATE_REQUEST_BODY = "validateRequestBody"
+    VALIDATE_REQUEST_PARAMETERS = "validateRequestParameters"
+    PARAM_NAME = "my-validator"
+    client = boto3.client("apigateway", region_name="us-west-2")
+    response = client.create_rest_api(name="my_api", description="this is my api")
+    api_id = response[ID]
+    response = client.create_request_validator(
+        restApiId=api_id,
+        name=PARAM_NAME,
+        validateRequestBody=True,
+        validateRequestParameters=True,
+    )
+
+    # test get single validator by
+    validator_id = response[ID]
+    response = client.get_request_validator(
+        restApiId=api_id, requestValidatorId=validator_id
+    )
+
+    response.pop("ResponseMetadata")
+
+    response.should.equal(
+        {
+            ID: validator_id,
+            NAME: PARAM_NAME,
+            VALIDATE_REQUEST_BODY: True,
+            VALIDATE_REQUEST_PARAMETERS: True,
+        }
+    )
+
+    # delete validator
+    response = client.delete_request_validator(
+        restApiId=api_id, requestValidatorId=validator_id
+    )
+
```

Comment:
We should have a more direct verification that this validator is now gone.
That can be as simple as verifying that this validator is no longer returned when requesting the full list:
```
validators = client.get_request_validators(restApiId=api_id)["items"]
validators.should.have.length_of(0)
```